app.js

```diff
@@ -48,9 +51,13 @@ app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 app.use(express.static(path.join(__dirname, 'app_client')));
 
-app.use('/', routes);
+// app.use('/', routes);
```

Comment:
@marcgardiner same issue mentioned above

Reply:
@igortb84 , same as above.

Reply:
just fixed it with this commit: https://github.com/marcgardiner/zi-spa/pull/6/commits/240599d9cd73aefaf8aedb6e6a7ca425ce710fb4